flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -133,7 +127,7 @@
 		.withDescription("A (semicolon-separated) list of patterns that specifies which classes should always be" +
 			" resolved through the plugin parent ClassLoader first. A pattern is a simple prefix that is checked " +
 			" against the fully qualified class name. This setting should generally not be modified. To add another " +
-			" pattern we recommend to use \"plugin.classloader.parent-first-patterns.additional\" instead.");
+			" pattern we recommend to use \"plugin.classloader.parent-first-patterns.GuavaFlinkConnectorRateLimiteradditional\" instead.");
```

Comment:
`GuavaFlinkConnectorRateLimiteradditional`

:)

Reply:
woooops, good catch. that would have been weird.